app/views/vaccinations/show.html.erb

```diff
@@ -23,7 +25,30 @@
   ) %>
 <% end %>
 
-<% if @vaccination_record&.administered? %>
+<% if @consent_response.nil? %>
+  <div class="nhsuk-card">
+    <div class="nhsuk-card__content">
+      <%= form_with model: @draft_consent_response,
```

Comment:
I wonder if it's correct that this should ever be pre-populated ... same issue with the other places where we have radio buttons on a patient page.

Reply:
Yeah, you're right that it could be prepopulated here which is probably unnecessary. :thinking: 

Let's kick this down the road